src/frontends/onnx/frontend/src/op/less_or_equal.cpp

```diff
@@ -0,0 +1,35 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "op/less_or_equal.hpp"
+OPENVINO_SUPPRESS_DEPRECATED_START
+
+#include "default_opset.hpp"
+
+namespace ngraph {
+namespace onnx_import {
+namespace op {
+namespace set_1 {
+OutputVector less_or_equal(const Node& node) {
+    const auto& input = node.get_ng_inputs();
```

Comment:
Looks like it also has a difference only in supported format. Could you add a check as we did before?
```cpp
    NGRAPH_CHECK(A.get_element_type() != ov::element::bf16 && B.get_element_type() != ov::element::bf16,
                 "The input data bfloat16 isn't supported in opset 12");
```